app/Models/Characters/Monster.php

```diff
@@ -46,8 +79,55 @@ public function getProficiencyBonusAttribute()
 
     function getSkillBonus(string $skill_name)
     {
-        $skill = $this->skills()->where('name', '=', $skill_name)->first();
+        $skill = $this->skills()->where('code', '=', $skill_name)->first();
```

Comment:
'='

Reply:
:o Didn't Laravel just add `firstWhere` or was that a dream?

Reply:
Yes, it is the dream